src/components/Overlay/Overlay.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  background: rgba(0, 0, 0, 0.7);
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 8000;
```

Comment:
I would put `z-index` to `theme` to make it easly managable, here it is just long shot